py/picca/tests/delta_extraction/expected_flux_tests.py

```diff
@@ -805,6 +812,442 @@ def test_expected_flux(self):
+        config.read_dict({
+            "data": {**desi_mock_data_kwargs, **{"wave solution": "log"}},
+            "expected flux": {
+                "type": "TrueContinuum",
+                "input directory": f"{THIS_DIR}/data",
+                "iter out prefix": "iter_out_prefix",
+                "out dir": f"{THIS_DIR}/results/",
+            },
+        })
+
+        for key, value in defaults_true_continuum.items():
+            if key not in config["expected flux"]:
+                config["expected flux"][key] = str(value)
+        for key, value in defaults_desisim_data.items():
+            if key not in config["data"]:
+                config["data"][key] = str(value)
+        data = DesisimMocks(config["data"])
+        expected_flux = TrueContinuum(config["expected flux"])
+
+        for forest in data.forests:
+            if forest.los_id == 59152:
+                expected_flux.read_true_continuum(forest)
+                if "REPLACE_TEST_VALUES" in os.environ and os.environ["REPLACE_TEST_VALUES"] == "Y":
+                    np.savetxt(data_dir / "true_true_cont_log.txt", forest.continuum)
+                np.testing.assert_almost_equal(
+                    forest.continuum,
+                    np.loadtxt(data_dir / "true_true_cont_log.txt")
+                )
+
+    def test_true_continuum_expected_flux(self):
+        """Test method compute expected flux for class TrueContinuum"""
+        # setup Forest variables; case: logarithmic wavelength solution
+        setup_forest("log", rebin=3)
+
+        data_dir = Path(__file__).parent / "data"
+
+        out_file = Path(__file__).parent / "results" / "Log" / "iter_out_prefix_compute_expected_flux_log.fits.gz"  
+        test_file = data_dir / "true_iter_out_prefix_compute_expected_flux_log.fits.gz"
+        
+        # initialize Data and Dr16ExpectedFlux instances
+        config = ConfigParser()
+        config.read_dict({
+            "data": {**desi_mock_data_kwargs, **{"wave solution": "log"}},
+            "expected flux": {
+                "type": "TrueContinuum",
+                "input directory": f"{THIS_DIR}/data",
+                "iter out prefix": "iter_out_prefix",
+                "out dir": f"{THIS_DIR}/results/",
+            },
+        })
+
+        for key, value in defaults_true_continuum.items():
+            if key not in config["expected flux"]:
+                config["expected flux"][key] = str(value)
+        for key, value in defaults_desisim_data.items():
+            if key not in config["data"]:
+                config["data"][key] = str(value)
+        data = DesisimMocks(config["data"])
+        expected_flux = TrueContinuum(config["expected flux"])
+
+        # compute the expected flux
+        expected_flux.compute_expected_flux(data.forests)
+
+        # check the results
+        for iteration in range(1,5):
+            if "REPLACE_TEST_VALUES" in os.environ and os.environ["REPLACE_TEST_VALUES"] == "Y":
```

Comment:
I would not add this copying thing here. The point of manually copying the files when we really need to update the test suite is to make sure we don't do it automatically by mistake (e.g. if the environment variable was set for some other reason)